documentation.yml

```diff
@@ -1,4 +1,6 @@
 toc:
+  - name: Aggregation
```

Comment:
Ah ok, I'm kind of the opposite, I think collect is actually very obvious what's happening and think it's a cool example :)

Anyway Im not too fussed about order so happy to revise

I was think about adding a `conversion` category for things like `flatten`,`explode`, and I also have a polyToLinestring almost ready and a viceversa, although I might add this category down the track when I've actually released a few more modules to fill it.